dev-games/openscenegraph-openmw/metadata.xml

```diff
@@ -16,10 +16,16 @@
 	<longdescription lang="en">
 		OpenSceneGraph on steroids. This fork was created to experiment with performance enhancements for OpenMW that are too controversial to be included in the general purpose OSG project - either for design reasons or backward compatibility reasons. OpenMW still works with "vanilla" OpenSceneGraph, but it is recommended to use the fork for better performance (around 10-15% on most devices). Note that the improvements are very specific to OpenMW and its animation code in particular; other OpenSceneGraph applications are not likely to see the same speed-up, or any speed-up at all if they are already GPU limited.
 	</longdescription>
+	<upstream>
+		<remote-id type="github">OpenMW/osg</remote-id>
+	</upstream>
```

Comment:
fwiw non-openmw openscenegraph put this below `</use>`. That's also what I usually do but it doesn't really matter, just saying given seem you want to keep those in sync where possible.